string device
int32 samples
float64 frequency
---
geometry_msgs/PoseStamped pose
